lib/utils/url_launcher.dart

```diff
@@ -206,13 +206,8 @@ class UrlLauncher {
     if (url != null) {
       if (PlatformInfos.isMobile) {
         await browser.open(
-          url: Uri.parse(url!),
-          options: ChromeSafariBrowserClassOptions(
-            android: AndroidChromeCustomTabsOptions(
-              shareState: CustomTabsShareState.SHARE_STATE_ON,
-            ),
-            ios: IOSSafariOptions(barCollapsingEnabled: true),
```

Comment:
Why delete these options? Any impact to `UiaRequestManager`?

Reply:
<img width="616" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/43041967/72b6752b-7be7-420b-92d7-241188443f5a">

Reply:
ok, but please make sure the same options with before and No impact to other functions, especially `UiaRequestManager`

Reply:
okay i will check the migration guide

Reply:
Done, i have changed to like before

Reply:
can't see any change 
why we need different open in app browser login for iOS and Android ?

Reply:
@drminh2807 the app bar, address bar, .... of CustomTab

Reply:
It supposes to have no change at all, the name is tricky, but it will open an inappwebview, because of the ios have to open with `registerIOS` to have cookie, other platforms can use this method